app/main.py

```diff
@@ -371,32 +399,179 @@ def update_live_alerts_data(n_intervals, ongoing_live_alerts, ongoing_frame_urls
     # And we deduce the subset of alerts that we can deem to be "live"
     live_alerts = all_alerts[mask].copy()
 
-    # Fetching live_alerts frames urls and instantiating a dict of live_alerts urls having event_id keys
-    dict_images_url_live_alerts = {}
+    # Some of these live alerts may have been blocked by the user, clicking on a "Ne plus voir cette alerte" button
+    # We filter these out thanks to the list of blocked_event_ids
+    live_alerts = live_alerts[
+        ~live_alerts['event_id'].isin(blocked_event_ids['event_ids'])
+    ].copy()
 
-    for _, row in live_alerts.iterrows():
+    # Is there any live alert to display?
+    if live_alerts.empty:
+        # If not, we do not update any of the callback's output
+        raise PreventUpdate
 
-        img_url = ""
+    else:
+        # If yes, there is a bit of work to do!
 
-        try:
-            img_url = api_client.get_media_url(row["media_id"]).json()["url"]
-        except Exception:
-            pass
+        # We load the data contained by the store_live_alerts_data dcc.Store component
+        temp = json.loads(ongoing_live_alerts)
+
+        # Has some alert data already been loaded so far?
+        if isinstance(temp, dict) and 'status' in temp.keys() and temp['status'] == 'never_loaded_alerts_data':
+            # If the dictionary loaded from the store_live_alerts_data dcc.Store component corresponds to the initial
+            # data attribute of the component, it means that alert data have never been loaded so far
```

Comment:
Nice way to check that the initial store_live_alerts_data has been fed with alert data in the first place !